src/Factory.php

```diff
@@ -290,7 +292,13 @@ final public static function isBooted(): bool
 
     final public static function faker(): Faker\Generator
     {
-        return self::configuration()->faker();
+        try {
+            return self::configuration()->faker();
+        } catch (FoundryNotBootedException $exception) {
+            throw new \RuntimeException(
+                "Cannot get Foundry's configuration. If using faker in a data provider, consider passing attributes as a callable."
```

Comment:
I'm wondering if this error could happen in an other case than in a data provider? :thinking: